### PR TITLE
Angular Signals support

### PR DESCRIPTION
**Pull request by titerman**

### CODE REVIEW

**Review by GoodDayForSurf — APPROVED** [review]

File: concepts/40 Angular Components/10 Getting Started/90 Support for Angular Signals.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Angular v16 and up includes a popular capability called "Angular Signals". It is a new, streamlined way to track changes in page data, and react to these changes.
```
Comment (albertov05) [inline concepts/40 Angular Components/10 Getting Started/90 Support for Angular Signals.md]:
```suggestion
Angular v16 and up include a popular capability called "Angular Signals". It is a new, streamlined way to track changes in page data, and react to these changes.
```

File: concepts/40 Angular Components/10 Getting Started/90 Support for Angular Signals.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+**Signals** are wrappers that listen to changes in page data. You can create two kinds of signals:
+
+* A *writable signal* wraps around a variable and fires when the variable's value changes. You can reference this signal throughout the application to manually overwrite the value of the underlying variable.
+* A *computed signal* is derivative. It uses internal logic to process data from other signals.
```
Comment (albertov05) [inline concepts/40 Angular Components/10 Getting Started/90 Support for Angular Signals.md]: What do you mean by "derivative"? Are you using this word as a noun in a mathematical context:
"the limit of the ratio of the change in a function to the corresponding change in its independent variable as the latter change approaches zero"

File: concepts/40 Angular Components/10 Getting Started/90 Support for Angular Signals.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+You can respond to signals with **effects**. Effects are functions that run when one or more signals fire.
+
+DevExtreme components are fully compatible with Angular Signals. If you want to integrate this capability into your application, you can examine a [sample DevExtreme application]( https://codesandbox.io/p/devbox/dx-signals-2m8hq8) with signals and effects.
+
+* The `prefix` signal in the example monitors the state of the "Prefix" dropdown. When the user changes the active option, the corresponding `effect` function filters the list of employees, and selects rows with employees that match the selected prefix.
+* The `selectedRows` signal monitors the state of the table and reports changes to the active row selection.
```
Comment (albertov05) [inline concepts/40 Angular Components/10 Getting Started/90 Support for Angular Signals.md]: What do you mean by "active" in the context of row selection?

File: concepts/40 Angular Components/10 Getting Started/90 Support for Angular Signals.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+Angular v16 and up include a popular capability called "Angular Signals". It is a new, streamlined way to track changes in page data, and react to these changes.
+
+**Signals** are wrappers that listen to changes in page data. You can create two kinds of signals:
+
+* A *writable signal* wraps around a variable and fires when the variable's value changes. You can reference this signal throughout the application to manually overwrite the value of the underlying variable.
+* A *computed signal* wraps around other signals and uses internal logic to process their data.
+
+You can respond to signals with **effects**. Effects are functions that run when one or more signals fire.
+
+DevExtreme components are fully compatible with Angular Signals. If you want to integrate this capability into your application, you can examine a sample DevExtreme application ) with signals and effects.
```
Comment (vladaskorohodova) [inline concepts/40 Angular Components/10 Getting Started/90 Support for Angular Signals.md]:
```suggestion
DevExtreme components are fully compatible with Angular Signals. If you want to integrate this capability into your application, you can examine a sample DevExtreme application with signals and effects.
```

**Pull request merged**